spec/views/categories/index_spec.rb

```diff
@@ -28,4 +28,4 @@
   #     end
   #     expect(current_path).to match group_expenses_path(@category)
   # end
```

Comment:
- [ ] For clean code and best practices, kindly ensure that you remove all the commented and used code lines from your code base